trackidledetector-processor/pom.xml

```diff
@@ -1,41 +1,43 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
 
-	<parent>
-		<groupId>com.esri.geoevent.parent</groupId>
-		<artifactId>track-idle-detector</artifactId>
-		<version>10.6.0</version>
-	</parent>
+  <parent>
+    <groupId>com.esri.geoevent.parent</groupId>
+    <artifactId>trackidledetector</artifactId>
+    <version>${revision}</version>
+  </parent>
 
-	<packaging>bundle</packaging>
-	<groupId>com.esri.geoevent.processor</groupId>
-	<artifactId>trackidledetector-processor</artifactId>
-	<name>Esri :: GeoEvent :: Processor :: Track Idle Detector</name>
-	
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.felix</groupId>
-				<artifactId>maven-bundle-plugin</artifactId>
-				<extensions>true</extensions>
-				<configuration>
-					<instructions>
-						<Bundle-SymbolicName>${project.groupId}.${project.artifactId}</Bundle-SymbolicName>
-						<Bundle-ContactAddress>geoevent@esri.com</Bundle-ContactAddress>
-						<Bundle-Version>${project.version}</Bundle-Version>
-						<Export-Package/>
-						<Private-Package>com.esri.geoevent.processor.trackidledetector</Private-Package>
-					</instructions>
-				</configuration>
-			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<configuration>
-					<source>1.8</source>
-					<target>1.8</target>
-				</configuration>
```

Comment:
Was that section removed or moved?

Reply:
Moved to parent.  Maven pom.xml best practices say to put as much as possible in the root/parent pom file under 'Management' sections then only use as necessary in the child pom(s).